src/main/java/com/hulu/ftl/annotations/Template.java

```diff
@@ -0,0 +1,29 @@
+package com.hulu.ftl.annotations;
+
+import java.util.Map;
+
+public class Template extends Annotation {
+    private String value;
+
+    public Template(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String getValue(Map map) {
+        String input = value;
+        for (Object item : map.entrySet()) {
```

Comment:
This could be written more efficiently, but I didn't want to bother with complicated regex stuff when I writing it